stl/base.py

```diff
@@ -89,6 +89,9 @@ def __init__(self, name):
   def __eq__(self, other):
     return NamedObject.__eq__(self, other) and self.params == other.params
 
+  def Resolve(self, env, resolved_params):
+      raise NotImplementedError('Resolved() is not needed: ' + self.name)
+
```

Comment:
It's an quirk of how pylint works. Pylint sees that NameObject.Resolve just does a `raise NotImplementedError`, so pylint considers that method to be abstract. Then it doesn't see an overrdie in the child class, so it throws an unimplemented abstract method warning.

Another option would be to just disable the warning in this file